Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.py

```diff
@@ -2197,6 +2286,338 @@ def list_dns_zone_recordsets(self, zone: str):
+                                   ) -> dict:
+        """
+            Update client list entries. Append, delete, and update entries in a single batch.
+        Args:
+            list_id: Unique identifier of client list.
+            account_switch_key: For customers who manage more than one account,
+                this runs the operation from another account. The Identity and Access Management
+                API provides a list of available account switch keys.
+
+        Returns:
+            <Response [200]>
+            Json response as dictionary
+        """
+
+        headers = {
+            "accept": "application/json",
+            "content-type": "application/json"
+        }
+        if account_switch_key == "":
+            url_suffix = f'client-list/v1/lists/{list_id}/items'
+        else:
+            url_suffix = f'client-list/v1/lists/{list_id}/items?accountSwitchKey={account_switch_key}'
+
+        return self._http_request(method='POST',
+                                  url_suffix=url_suffix,
+                                  headers=headers
+                                  )
+
+    def activate_a_client_list(self,
+                               list_id: str,
+                               action: str,
+                               comments: str,
+                               network: str,
+                               notification_recipients: list,
+                               account_switch_key: str = "",
+                               siebel_ticketid: str = "",
+                               ) -> dict:
+        """
+            Activates a client list on the staging or production network.
+        Args:
+            list_id: Unique identifier of client list.
+            account_switch_key: For customers who manage more than one account,
+                this runs the operation from another account. The Identity and Access Management
+                API provides a list of available account switch keys.
+            action: Actions you can take for a client list: either ACTIVATE or DEACTIVATE.
+            comments: A brief description for the activation.
+            network: The network environment where you activate your client list: either STAGING or PRODUCTION.
+            notification_recipients: Users to notify via email.
+            siebel_ticketid: Identifies the Siebel ticket, if the activation is linked to one.
+
+        Returns:
+            <Response [200]>
+            Json response as dictionary
+        """
+
+        headers = {
+            "accept": "application/json",
+            "content-type": "application/json"
+        }
+
+        pre_body = {
+            "action": action,
+            "network": network,
+            "comments": comments,
+            "notificationRecipients": notification_recipients,
+            "siebelTicketId": siebel_ticketid
+        }
+
+        body = filter_empty_values(pre_body)
```

Comment:
```suggestion
        remove_nulls_from_dictionary(body)
```
Use the `remove_nulls_from_dictionary()` function from CommonServerPython